03_analysis/compare_ffs.py

```diff
@@ -496,26 +570,31 @@ def label(x, color, label):
 
     # Remove axes details that don't play well with overlap
     g.set_titles("")
-    g.set(yticks=[])
-    g.despine(bottom=True, left=True)
+#    g.set(yticks=[])
+    g.despine(bottom=True)#, left=True)
+    # ax = plt.gca()
+    # ax.spines['left'].set_visible(True)
+    # ax.spines['left'].set_position('zero')
+    # ax.set_yticks([0.4])
     if what_for == 'paper':
-        plt.gcf().set_size_inches(5, 3)
+        plt.gcf().set_size_inches(7, 3)
     elif what_for == 'talk':
         plt.gcf().set_size_inches(12, 9)
 
     # adjust font sizes
     plt.xlabel(x_label, fontsize=ridgedict["xfontsize"])
+    plt.ylabel('Density', fontsize=ridgedict["xfontsize"])
     plt.xticks(fontsize=ridgedict["xfontsize"])
 
 
     # save with transparency for overlapping plots
-    plt.savefig(out_file, bbox_inches='tight', transparent=True)
+    plt.savefig(out_file, transparent=True, bbox_inches='tight')
     plt.clf()
     #plt.show()
 
 
 def draw_density2d(x_data, y_data, title, x_label, y_label, out_file, what_for='talk',
-    bins=20, x_range=None, y_range=None, z_range=None, z_interp=True):
+                   bins=20, x_range=None, y_range=None, z_range=None, z_interp=True, symlog=False):
```

Comment:
Let's `symlog` to the docstring ("Bool", "True to represent y-axis on (symmetric) log scale; False for linear y-scaling")

Reply:
Resolved in f0d5033